### DISCUSSION TITLE
[Feature] please let me filter on photos that are missing position

### DISCUSSION THREAD

**Discussion by AndKe** [immich-app/immich#13291] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Please add an option so that I could see only photos that are NOT geolocated, so I can geolocate them manually.

![image](https://github.com/user-attachments/assets/b40a3fb7-396d-4e2a-8b5e-207ddcd4ea98)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This is already possible, by selecting "unknown" for the country filter
